### PR TITLE
#3: Updated USNG and added unit tests

### PR DESCRIPTION
**Pull request by AzGoalie**
Updated USNG to use proper precision and normalized distances. 
Added unit tests to test for precision and bounding boxes. 
@harrison-tarr @mverret @stustison @brendan-hofmann

### CODE REVIEW

**Comment by AzGoalie** [comment]
Still converting old tests to new precision.

File: tests/tests.js [hunk 1]
```diff
@@ -452,27 +475,35 @@ describe('Convert Lat/Lon Bounding Box to USNG', function(){
...
   });
+<<<<<<< HEAD
   describe('around Toliara city center in Madagascar', function(){
     it('should return 38K LV 64 17', function(){
-      chai.assert.equal("38K LV 64 17", converter.LLBboxtoUSNG(-23.31, -23.39, 43.71, 43.63));
+      chai.assert.equal("38K LV 66 12", converter.LLBboxtoUSNG(-23.395, -23.39, 43.70, 43.695));
     });
   });
+=======
+//  describe('around Toliara city center in Madagascar', function(){
```
Comment (brendan-hofmann) [inline tests/tests.js]: Commented out code: fix or remove?

**Comment by harrison-tarr** [comment]
:+1: lgtm

**Comment by mverret** [comment]
:+1:

**Comment by brendan-hofmann** [comment]
:+1:

**Comment by harrison-tarr** [comment]
does the USNG need to be republished on bower or anything now? is it automatic?

**Pull request merged**